.tutone.yml

```diff
@@ -475,6 +475,12 @@ packages:
         argument_type_overrides:
           accountId: "Int!"
           accounts: "[CloudUnlinkAccountsInput!]!"
+      - name: cloudUpdateAccount
+        max_query_field_depth: 3
+        exclude_fields:
+          - "integration"
+          - "integrations"
+          - "provider"
```

Comment:
why this 🤔

Reply:
in the response `linkedAccounts` some arguments have mandatory fields. Due to which I was getting error `expected int but got nil`( Please refer to the image- id field is mandatory for the `integration` argument )
As we don't require any field from these arguments for the update functionality so excluded it! 
![image](https://github.com/user-attachments/assets/b8db5a5e-64b1-4ec2-b7ef-dc05336c26bb)



Please note we are saving only the below fields in the terraform state.
<img width="398" alt="image" src="https://github.com/user-attachments/assets/c90ca73e-f504-43cf-a8c5-af081e8ced15">